tools/metadata_viewer/kvstore.py

```diff
@@ -243,11 +264,33 @@ def decode_storage_value(type, v):
     return ret
 
 
+def read_peristed_batch(rdr):
```

Comment:
nit: I think this name is a bit misleading for a top-level function, so it is better to move this inside `decode_offset_translator_value`